README.md

```diff
@@ -37,17 +37,22 @@ COPY . /startup/
 
 The following environment variables can be set to change the Couchbase Server configuration:
 
-| Env Variable    | Description                                                    |
-| ------------    | -----------                                                    |
-| CB_DATARAM      | Data service RAM in megabytes, default `512`                   |
-| CB_INDEXRAM     | Index service RAM in megabytes, default `256`                  |
-| CB_SEARCHRAM    | Search (FTS) service RAM in megabytes, default `256`           |
-| CB_SERVICES     | Services to enable, default `kv,n1ql,index,fts`                |
-| CB_INDEXSTORAGE | Index storage mode, `forestdb` (default) or `memory_optimized` |
-| CB_USERNAME     | Couchbase user name, default `Administrator`                   |
-| CB_PASSWORD     | Couchbase password, default `password`                         |
-
-Values for CB_SERVICES and CB_INDEXSTORAGE correspond to parameters for the [Couchbase REST API](https://docs.couchbase.com/server/current/rest-api/rest-bucket-create.html).
+| Env Variable    | Description                                                                                |
+| ------------    | -----------                                                                                |
+| CB_CLUSTER_NAME | Specify the name of the cluster                                                            |
+| CB_DATARAM      | Data service RAM in megabytes, default `512`                                               |
+| CB_INDEXRAM     | Index service RAM in megabytes, default `256`                                              |
+| CB_SEARCHRAM    | Search (FTS) service RAM in megabytes, default `256`                                       |
+| CB_ANALYTICSRAM | Analytics service RAM in megabytes. Only applicable if `cbas` is added to `CB_SERVICES`    |
+| CB_EVENTINGRAM  | Eventing service RAM in megabytes. Only applicable if `eventing` is added to `CB_SERVICES` |
+| CB_SERVICES     | Services to enable, default `kv,n1ql,index,fts`                                            |
+| CB_INDEXSTORAGE | Index storage mode, `forestdb` (default) or `memory_optimized`                             |
+| CB_USERNAME     | Couchbase user name, default `Administrator`                                               |
+| CB_PASSWORD     | Couchbase password, default `password`                                                     |
+
+Values for CB_SERVICES and CB_INDEXSTORAGE correspond to parameters for the [Couchbase REST API](https://docs.couchbase.com/server/current/rest-api/rest-node-provisioning.html).
+
+**NOTE:** If you configure `CB_SERVICES` to create the `cbas` analytics service, make sure you set `CB_ANALYTICSRAM` to a minimum of `1024`.
```

Comment:
It just occurred to me, perhaps we should put a default value for CB_ANALYTICSRAM and CB_EVENTINGRAM in the Dockerfile?